reference/src/lib/translations.js

```diff
@@ -10,6 +10,7 @@ export default {
 			title: 'Your privacy settings',
 			description: 'We use cookies to bring you personalized content, relevant ads, social media features, and to better understand how you use our website. To do this, we sometimes share this data with social media, advertising, and analytics partners, who may in turn combine it with other data you\'ve given them.<br>Click the "SETTINGS" button to change your privacy settings and learn more about our use of cookies, or click “ACCEPT” to agree with the above mentioned.<br>You can change your mind and change your consent choices at any time by returning to this site.',
 			acceptAll: 'Accept',
+            privacyPolicy: 'Privacy policy',
```

Comment:
tabbing :(

Reply:
Thanks, sorry. (btw, I really hate that we don't use spaces 😅)